qa/covid19_qa/elasticsearch_qa.py

```diff
@@ -8,20 +8,43 @@
 ES_HOST = os.getenv("ES_HOST")
 es = Elasticsearch([{"host": ES_HOST, "port": os.environ["ES_PORT"]}]) if ES_HOST else None
 
+DEFAULT_ES_QUERY_CONF = [
+    {"operator": "and", "minimum_should_match": 80, "fuzziness": 1, "boost": 3},
+    {"operator": "and", "minimum_should_match": 60, "fuzziness": 1, "boost": 4},
+    {"operator": "or", "minimum_should_match": 80, "fuzziness": 1, "boost": 3},
+    {"operator": "or", "minimum_should_match": 50, "fuzziness": 1, "boost": 1},
+]
 
-def search_query_string(query: str, index_pattern: str = "covid*", size: int = 10, fragments: int = 1,
-                        fragment_size: int = 700, text_field: str = "article_text") -> Iterator[Mapping[str, Any]]:
+
+def get_match_query_from_conf(query: str, text_field: str, conf: Dict) -> Dict:
+    return {
+        "match": {
+            text_field: {
+                "query": query,
+                "operator": conf.get("operator"),
+                "minimum_should_match": f"{conf.get('minimum_should_match')}%",
+                "fuzziness": conf.get("fuzziness"),
+                "boost": conf.get("boost")
+            }
+        }
+    }
+
+
+def search_query_string(query: str, index_pattern: str = "covid*", size: int = 20, fragments: int = 2,
+                        fragment_size: int = 1500, text_field: str = "article_text",
+                        es_query_configurations: Iterator[Dict] = DEFAULT_ES_QUERY_CONF) -> Iterator[Mapping[str, Any]]:
+    es_query = query.replace("?", "").replace("¿", "")
```

Comment:
Given that there's a non-ASCII char here, we should declare the encoding of this Python file, cause otherwise, it could be troublesome to run.

Reply:
good catch